Add ignore-commands config option to Redis Raft (#188)
RedisLabs/redisraft: docs/Deployment.md

of foreign shardgroup clusters.

*Default: 5000*

### `ignored-commands`

> A comma sepeated list of additional commands that RedisRaft should not intercept and append to the Raft log before executing.

Reviewer: ```suggestion
A comma separated list of additional commands that RedisRaft should not intercept and append to the Raft log before executing.
```

> Im general this is useful when used with other modules that don't wnat some or all of their commands handled via raft.

Reviewer 2: ```suggestion
Im general this is useful when used with other modules that don't want some or all of their commands handled via raft.
```

> Im general this is useful when used with other modules that don't wnat some or all of their commands handled via raft.

Reviewer: ```suggestion
In general this is useful when used with other modules that don't want some or all of their commands handled via raft.
```

> *Example*: ignore,mycommand

Reviewer: For a moment, I thought it should start with "ignore,xxx" keyword, this is not the case right ? :) Maybe use another word instead 'ignore' here, something like command1,command2,command3

> By default, this configuration option will be mepty and no additional commands will be ignored beyond those RedisRaft is hard coded to ignore.

Reviewer 2: ```suggestion
By default, this configuration option will be empty and no additional commands will be ignored beyond those RedisRaft is hard coded to ignore.
```

> By default, this configuration option will be mepty and no additional commands will be ignored beyond those RedisRaft is hard coded to ignore.

Reviewer: ```suggestion
By default, this configuration option will be empty and no additional commands will be ignored beyond those RedisRaft is hard coded to ignore.
```
